pkg/envtest/helper.go

```diff
@@ -57,21 +66,19 @@ func existsUnstructured(s1, s2 []*unstructured.Unstructured) bool {
 	return false
 }
 
-func runtimeListToUnstructured(l []runtime.Object) []*unstructured.Unstructured {
+func runtimeCRDListToUnstructured(l []runtime.Object) []*unstructured.Unstructured {
 	res := []*unstructured.Unstructured{}
 	for _, obj := range l {
-		m, err := runtime.DefaultUnstructuredConverter.ToUnstructured(obj.DeepCopyObject())
-		if err != nil {
+		u := &unstructured.Unstructured{}
+		if err := crdScheme.Convert(obj, u, nil); err != nil {
```

Comment:
should we at least log something here? we're pretty much just silently skipping resources here, right?